app/projects/[id]/page.tsx

```diff
@@ -32,7 +32,6 @@ export default function ProjectDetailPage() {
     (project) => String(project.id) === router.split("/").slice(1)[1]
   )[0]
 
-  const githubLink = findProject.links.github
   const twitterLink = findProject.links.twitter
```

Comment:
Not sure why these variables are here in the first place, but you should be consistent in your usage. Meaning you should be using githubLink, twitterLink, websiteLink from here on out. This is not currently so below. Also `const {twitter: twitterLink, websiteLink, github:githubLink} = findProject.links`  would yield same results in one line